dbt/include/snowflake/macros/adapters.sql

```diff
@@ -92,6 +94,10 @@
   );
 {% endmacro %}
 
+{% macro snowflake__create_view_as(relation, sql) -%}
+  {{ snowflake__create_view_as_with_temp_flag(False, relation, sql) }}
```

Comment:
Part 2 of 3 of multi-part suggestion:

```suggestion
  {{ snowflake__create_view_as_with_temp_flag(relation, sql, temporary=False) }}
```